ibm_catalog.json

```diff
@@ -9,52 +9,50 @@
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/blob/main/solutions/standard/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-client-to-site-vpn/main/images/c2s_vpn.svg",
       "provider_name": "IBM",
       "features": [
         {
-          "description": "Supports creating and configuring a new Secrets Manager instance, or using an existing one, to create a secret group and a new private cert.",
-          "title": "Creates and configures an optional Secrets Manager instance, or uses existing."
+          "description": "Supports using an existing Secrets Manager instance, to create a secret group and a new private cert.",
+          "title": "Configures existing Secrets Manager instance, and create a secret group and a new private cert."
         },
         {
-          "description": "The network ACL on this subnet grants all access from any source.",
-          "title": "A subnet named 'client-to-site-subnet' in the landing-zone management VPC"
+          "description": "The network ACL on this subnet grants the access from sources according to the rules defined with 'network_acls' input variable.",
+          "title": "A subnet named 'client-to-site-subnet' in the VPC"
         },
         {
-          "description": "A new security group named 'client-to-site-sg' that allows all incoming request from any source",
+          "description": "A new security group named 'client-to-site-sg' that allows incoming request from sources defined in'security_group_rules'",
           "title": "client-to-site Security Group"
         },
         {
           "description": "An IAM access group allowing users to authenticate and connect to the client-to-site VPN gateway",
           "title": "A new IAM Access Group for VPN users"
         },
         {
-          "description": "VPN gateway located in the client-to-site-subnet subnet with routes configured to allow accessing the landing zone VPCs.",
+          "description": "VPN gateway located in the client-to-site-subnet subnet with routes configured to allow accessing the VPCs.",
           "title": "A client-to-site VPN server"
         }
       ],
       "flavors": [
         {
-          "label": "Standard",
-          "name": "standard",
-          "install_type": "extension",
-          "working_directory": "extensions/landing-zone",
+          "label": "Quickstart",
+          "name": "quickstart",
+          "install_type": "fullstack",
+          "working_directory": "solutions/quickstart",
```

Comment:
@akocbek Can you add `"dependency_version_2": true` here please? Catalog team are asking us to use this

Reply:
(same way I did in https://github.com/terraform-ibm-modules/terraform-ibm-cos/pull/789)

Reply:
done